var/spack/repos/builtin/packages/mesa/package.py

```diff
@@ -8,41 +8,34 @@
 import sys
 
 
-class Mesa(AutotoolsPackage):
+class Mesa(MesonPackage):
     """Mesa is an open-source implementation of the OpenGL specification
      - a system for rendering interactive 3D graphics."""
 
     homepage = "http://www.mesa3d.org"
-    maintainers = ['v-dobrev']
+    maintainers = ['chuckatkins']
```

Comment:
Why not have both maintainers?

Reply:
Sure.  This seemed like an entirely new package, but it certainly makes sense to have both.

Reply:
Same software, just different build system. @v-dobrev could have chosen to be a maintainer because he was familiar with how the recipe was written, or more likely because he cares about `mesa` and wants to make sure it doesn't break. Unless he says otherwise, I tend to err on the side of "more maintainers is always better" 😄